### ISSUE TITLE
Bot down?

### ISSUE THREAD

**Issue by Bluscream** [asonix/telegram-event-bot#2]
![image](https://user-images.githubusercontent.com/3318223/67905763-57397800-fb6a-11e9-9f6c-3e4718998c8b.png)

**Comment by asonix** [comment]
Yes, this bot is currently down. I'm going to be rewriting this project at some point and maybe I'll put the bot up again afterwards

**Comment by Bluscream** [comment]
I know it's a lot of fucking work

![image](https://user-images.githubusercontent.com/3318223/67908462-86081c00-fb73-11e9-8dd6-5d1ab63b7744.png)
